libbeat/_meta/config.yml.tmpl

```diff
@@ -90,12 +90,19 @@ output.elasticsearch:
 {{if not .UseObserverProcessor}}
 # Configure processors to enhance or manipulate events generated by the beat.
 
+{{if .UseKubernetesMetadataProcessor}}
 processors:
   - add_host_metadata: ~
   - add_cloud_metadata: ~
   - add_docker_metadata: ~
   - add_kubernetes_metadata: ~
 {{else}}
+processors:
+  - add_host_metadata: ~
+  - add_cloud_metadata: ~
+  - add_docker_metadata: ~
+{{end}}
```

Comment:
could this change  be just adding the kubernetes processor within the existing list? Something like:

```
processors:
  - add_host_metadata: ~
  - add_cloud_metadata: ~
  - add_docker_metadata: ~
{{if .UseKubernetesMetadataProcessor}}
  - add_kubernetes_metadata: ~
{{end}}
```

Reply:
That was what I tried first, however this adds extra blank lines because of the template conditionals like:

```
processors:
  - add_host_metadata: ~
  - add_cloud_metadata: ~
  - add_docker_metadata: ~

  - add_kubernetes_metadata: ~

```
If this is ok, I can change it.

Reply:
gotcha, try with `{{- if -}}`!